test/TestCBOR.sol

```diff
@@ -13,6 +13,21 @@ contract TestCBOR {
   event Log(string _topic, uint256 _value);
   event Log(string _topic, bytes _value);
 
+  function testBoolDecode() external {
+    bool decodedFalse = CBOR.valueFromBytes(hex"f4").decodeBool();
+    bool decodedTrue = CBOR.valueFromBytes(hex"f5").decodeBool();
+    Assert.equal(
+      decodedFalse,
+      false,
+      "CBOR-encoded false value should be decoded into a CBOR.Value containing the correct bool false value"
+    );
+    Assert.equal(
+      decodedTrue,
+      true,
+      "CBOR-encoded true value should be decoded into a CBOR.Value containing the correct bool true value"
+    );
+  }
```

Comment:
Try to also test `0xf6` for error.

Reply:
None of the CBOR reverts were tested, but we'll add it! :P

Reply:
I suppose it wasn't done before because it is not the most "friendly" thing to do in a Solidity test (see https://www.trufflesuite.com/docs/truffle/testing/writing-tests-in-solidity).